develop/tutorials/articles/02-from-liferay-6-to-liferay-7/07-upgrading-plugins-to-liferay-7/05-upgrading-themes.markdown

```diff
@@ -108,60 +116,70 @@ them now.
                 <compatibility>
                         <version>7.0.0+</version>
                 </compatibility>
+
                 ...
+
         </look-and-feel>
 
-3.  If your theme uses the Themes Generator and you haven't already run the
-    `gulp upgrade` task, open its `package.json` file and update the Liferay
-    version references to `7.0`.
+3.  If your theme uses the Themes Generator and has a `package.json` file,
+    update the file's Liferay version references to `7.0`.
 
 Your theme's Liferay version references are updated for Liferay 7. Next, you'll
 update the CSS.
 
 ## Updating the CSS [](id=updating-the-css)
 
-As a result of Liferay 7's UI improvements some CSS files have been removed and
+As a result of Liferay 7's UI improvements, some CSS files have been removed and
 others added. The directory structure has changed, as well. Liferay 7's update
-from Bootstrap 2 to Bootstrap 3 requires changes to CSS classes variables.
-Finally, the CSS responsiveness tokens have changed. Conforming to all these
-changes is part of the upgrade process.
+from Bootstrap 2 to Bootstrap 3 requires CSS classes variable changes. Lastly,
+the CSS responsiveness tokens have changed. Conforming to all these changes is
+part of the upgrade process.
 
-In this section, you'll update your theme's CSS. You'll start with updating CSS
-file names.
+In this section, you'll update your theme's CSS. You can start with updating CSS
+file names for Sass.
 
 ### Updating CSS File Names for Sass [](id=updating-css-file-names-for-sass)
 
-Liferay 7 themes should follow the Sass naming convention.
+Liferay 7 expects themes to use the Sass extension to CSS. Some of the CSS must
+be renamed to follow standard Sass file naming conventions. 
 
 +$$$
 
-**Note:** The `gulp upgrade` task updates the names automatically.
+**Note:** The `gulp upgrade` task renames the files automatically.
 
 $$$
 
-To follow the Sass naming convention, rename all the CSS files in your
-`_diffs/css` folder, except `main.scss` and `aui.scss`.
+For each CSS file you've modified from the base theme, except `main.scss` and
+`aui.scss`, change its suffix from `.css` to `.sccs`. 
+
+Then, prepend an underscore (`_`) to the name of all Sass partial files. 
 
 For example, rename `custom.css` to `_custom.scss`.
 
+Refer to the [Theme Reference Guide](/develop/reference/-/knowledge_base/7-0/theme-reference-guide)
+for a complete list of expected names for theme CSS files. 
+
 Here are the Lunar Resort theme's modified CSS files with updated names:
 
-- `css/`
-    - `_aui_variables.scss`
-    - `_custom.scss`
+- `_diffs/`
```

Comment:
Ah, right. I thought about that but didn't circle back to it.

I think switching it back to just css/ makes sense, but maybe note in the text above that this would be _diffs/css/ in a Plugins SDK theme.